src/utils/dom.js

```diff
@@ -78,3 +79,18 @@ export function focus(root: ?HTMLElement, selector?: string, focusRoot: boolean
         root.focus();
     }
 }
+
+export function scrollItemIntoView(id: ?string, className: string): void {
```

Comment:
I don't think we should introduce a new name i.e. "scroll _item_ into view". Why not make it consistent?